CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.1...3.16)
+cmake_minimum_required(VERSION 3.16)
```

Comment:
To be too restrictive limits the usability. I would like to support platforms which have old toolset too.
What problem it caused for you?